ds/data/processed/eurostat/epo_patent_applications.nuts2.csv

```diff
@@ -1,736 +1,841 @@
-2000,UKJ1,2010,547
-2000,UKJ2,2010,396
-2000,UKJ3,2010,188
-2000,UKJ4,2010,89
-2000,UKK1,2010,208
-2000,UKK2,2010,65
-2000,UKK3,2010,18
-2000,UKK4,2010,36
-2000,UKL1,2010,46
-2000,UKL2,2010,61
-2000,UKM2,2010,166
-2000,UKM3,2010,128
-2000,UKM5,2010,22
-2000,UKM6,2010,4
-2000,UKN0,2010,45
-1999,UKC1,2010,20
-1999,UKC2,2010,26
-1999,UKD1,2010,25
-1999,UKD3,2010,160
-1999,UKD4,2010,56
-1999,UKD6,2010,64
-1999,UKD7,2010,50
-1999,UKE1,2010,37
-1999,UKE2,2010,39
-1999,UKE3,2010,52
-1999,UKE4,2010,118
-1999,UKF1,2010,111
-1999,UKF2,2010,124
-1999,UKF3,2010,13
-1999,UKG1,2010,83
-1999,UKG2,2010,51
-1999,UKG3,2010,147
-1999,UKH1,2010,187
-1999,UKH2,2010,189
-1999,UKH3,2010,73
-1999,UKJ1,2010,388
-1999,UKJ2,2010,269
-1999,UKJ3,2010,118
-1999,UKJ4,2010,79
-1999,UKK1,2010,142
-1999,UKK2,2010,51
-1999,UKK3,2010,20
-1999,UKK4,2010,32
-1999,UKL1,2010,37
-1999,UKL2,2010,33
-1999,UKM2,2010,108
-1999,UKM3,2010,64
-1999,UKM5,2010,20
-1999,UKM6,2010,11
-1999,UKN0,2010,35
-1998,UKC1,2010,11
-1998,UKC2,2010,25
-1998,UKD1,2010,11
-1998,UKD3,2010,126
-1998,UKD4,2010,54
-1998,UKD6,2010,46
-1998,UKD7,2010,50
-1998,UKE1,2010,37
-1998,UKE2,2010,25
-1998,UKE3,2010,48
-1998,UKE4,2010,129
-1998,UKF1,2010,102
-1998,UKF2,2010,89
-1998,UKF3,2010,15
-1998,UKG1,2010,59
-1998,UKG2,2010,55
-1998,UKG3,2010,120
-1998,UKH1,2010,159
-1998,UKH2,2010,132
-1998,UKH3,2010,62
-1998,UKJ1,2010,325
-1998,UKJ2,2010,204
-1998,UKJ3,2010,102
-1998,UKJ4,2010,67
-1998,UKK1,2010,97
-1998,UKK2,2010,37
-1998,UKK3,2010,14
-1998,UKK4,2010,8
-1998,UKL1,2010,15
-1998,UKL2,2010,25
-1998,UKM2,2010,119
-1998,UKM3,2010,70
-1998,UKM5,2010,7
-1998,UKM6,2010,3
-1998,UKN0,2010,21
-1997,UKC1,2010,8
-1997,UKC2,2010,17
-1997,UKD1,2010,17
-1997,UKD3,2010,130
-1997,UKD4,2010,52
-1997,UKD6,2010,41
-1997,UKD7,2010,34
-1997,UKE1,2010,24
-1997,UKE2,2010,22
-1997,UKE3,2010,37
-1997,UKE4,2010,79
-1997,UKF1,2010,90
-1997,UKF2,2010,70
-1997,UKF3,2010,9
-1997,UKG1,2010,48
-1997,UKG2,2010,51
-1997,UKG3,2010,116
-1997,UKH1,2010,120
-1997,UKH2,2010,164
-1997,UKH3,2010,40
-1997,UKJ1,2010,277
-1997,UKJ2,2010,203
-1997,UKJ3,2010,91
-1997,UKJ4,2010,49
-1997,UKK1,2010,95
-1997,UKK2,2010,26
-1997,UKK3,2010,9
-1997,UKK4,2010,11
-1997,UKL1,2010,31
-1997,UKL2,2010,19
-1997,UKM2,2010,66
-1997,UKM3,2010,68
-1997,UKM5,2010,6
-1997,UKM6,2010,4
-1997,UKN0,2010,25
-1996,UKC1,2010,20
-1996,UKC2,2010,20
-1996,UKD1,2010,19
-1996,UKD3,2010,189
-1996,UKD4,2010,66
-1996,UKD6,2010,54
-1996,UKD7,2010,49
-1996,UKE1,2010,31
-1996,UKE2,2010,59
-1996,UKE3,2010,55
-1996,UKE4,2010,142
-1996,UKF1,2010,118
-1996,UKF2,2010,183
-1996,UKF3,2010,18
-1996,UKG1,2010,102
-1996,UKG2,2010,76
-1996,UKG3,2010,177
-1996,UKH1,2010,181
-1996,UKH2,2010,262
-1996,UKH3,2010,54
-1996,UKJ1,2010,395
-1996,UKJ2,2010,262
-1996,UKJ3,2010,98
-1996,UKJ4,2010,84
-1996,UKK1,2010,111
-1996,UKK2,2010,63
-1996,UKK3,2010,6
-1996,UKK4,2010,34
-1996,UKL1,2010,25
-1996,UKL2,2010,33
-1996,UKM2,2010,205
-1996,UKM3,2010,130
-1996,UKM5,2010,16
-1996,UKM6,2010,30
-1996,UKN0,2010,53
+2016,UKC1,2013,30
```

Comment:
Shouldn't the spec year here be 2016? (*)

Reply:
No - this uses fixed NUTS categories. 2010 for patents and 2013 for trademarks.

Reply:
Ah right that's why we added the `warning`, never mind, sorry for the noise